### ISSUE TITLE
Pragma problems

### ISSUE THREAD

**Issue by pgulutzan** [tarantool/tarantool#3541]
I have Ubuntu 14.04. I have Tarantool 2.0, pulled from source today.

I execute this SQL statement:
```
pragma schema_version;
```
Result:
```
tarantool: /home/pgulutzan/tarantool-2.0/src/box/sql/vdbe.c:1393: sqlite3VdbeExec: Assertion `pOp->p1>0' failed.
Aborted (core dumped)
```

I execute these SQL statements:
```
CREATE TABLE W1 (s1 INT PRIMARY KEY);
CREATE TABLE W2 (s1 INT PRIMARY KEY, s2, FOREIGN KEY (s2) REFERENCES W1 (s1));
PRAGMA foreign_keys (false);
INSERT INTO W2 VALUES (0,0);
PRAGMA foreign_key_check ('W2');
```
Result:
```
tarantool> tarantool: /home/pgulutzan/tarantool-2.0/src/box/sql/vdbe.c:1445: sqlite3VdbeExec: Assertion `((&pMem[i])->flags & 0x0100)==0' failed.
Aborted (core dumped)
```

I execute this SQL statement:
```
PRAGMA sql_default_engine('MEMTX');
```
Result:
No error, so perhaps this will be called "not a bug".
But in NoSQL {engine='MEMTX'} would not work.
And in other PRAGMA statements, for example
```
PRAGMA table_info('T');
```
the argument is case sensitive -- table 't' won't be found.
So consider this a feature request for a consistent rule.

**Comment by kyukhin** [comment]
Second point will be solved w/ patch for FKs refactoring.
Third point (about case) I wouldn't treat as a bug at all.

So, I'll close it when FKs refactoring patchset will hit 2.0 branch.

**Comment by Korablev77** [comment]
Closed by deea069324f141c6ef4f80b58ba828a0aef9ce91: we got rid off `pragma foreign_key_check`.
